variables.tf

```diff
@@ -308,3 +308,225 @@ variable "check_ec2_imdsv2" {
   type        = bool
   default     = false
 }
+
+variable "check_access_keys_rotated" {
+  description = "Enable access-keys-rotated rule"
```

Comment:
How does this rule work? Keys have been rotated once? Been rotated more than once? Rotated frequently? The description doesn't help me understand what this config rule is checking exactly.

Reply:
[access-keys-rotated](https://docs.aws.amazon.com/config/latest/developerguide/access-keys-rotated.html)

Checks if active access keys are rotated. There is another variable that is a parameter is this rule `maxAccessKeyAge` with a default of 90 days `Maximum number of days without rotation.`